site/pages/talks.rst

```diff
@@ -1,10 +1,622 @@
+
 .. title: Talks
 .. slug: talks
-.. date: 2019-02-17 12:20:00 UTC+07:00
-.. tags: draft
+.. date: 2019-05-16 12:46:19 UTC+07:00
+.. tags:
 .. category:
 .. link:
-.. description:
-.. type: text
+.. description: List of confirmed talks.
+.. type: macro
```

Comment:
This page is not being parsed as markdown, see https://deploy-preview-187--serene-mestorf-5c91d0.netlify.com/en/talks/

I guess this is because `macro` is not a valid type: https://getnikola.com/handbook.html#post-types